src/app/loading.module.css

```
@@ -0,0 +1,38 @@
+.loading {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  backdrop-filter: blur(3px);
+
+  .particles {
+    position: relative;
+    width: 100px;
+    height: 100px;
+
+    .particle {
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      width: 10px;
+      height: 10px;
+      background-color: var(--color-gray-10);
+      border-radius: 50%;
```

بهتره به جاش از یه مقدار زیاد استفاده کنید که اگه احیاناً Width و Height برابر نبودن، بازم خوب نمایش داده بشه

```suggestion
      border-radius: 999rem;
```